husky_components/datagrid/main.js

```diff
@@ -1084,6 +1086,37 @@
             },
 
             /**
+             * Parses data and returns normalized embedded items.
+             *
+             * @param data
+             * @param parent
+             */
+            parseEmbedded: function(data, parent) {
```

Comment:
What purpose does this have? Does it create a tree, like what is done until now for the categories? Maybe call it `parseTree` instead, so that it is more clear?